lib/config/presets/local/index.ts

```diff
@@ -9,12 +9,12 @@ import * as gitlab from '../gitlab';
 import type { Preset, PresetConfig } from '../types';
 
 const resolvers = {
-  [PlatformId.Azure]: azure,
-  [PlatformId.Bitbucket]: bitbucket,
-  [PlatformId.BitbucketServer]: bitbucketServer,
-  [PlatformId.Gitea]: gitea,
-  [PlatformId.Github]: github,
-  [PlatformId.Gitlab]: gitlab,
+  azure,
+  bitbucket,
+  ['bitbucket-server']: bitbucketServer,
```

Comment:
```suggestion
  'bitbucket-server': bitbucketServer,
```
shouldn't this work too? 🤔